internal/fs/fs.go

```diff
@@ -224,18 +224,6 @@ func createFileCacheHandler(cfg *ServerConfig) (fileCacheHandler *file.CacheHand
 	fileInfoCache := lru.NewCache(sizeInBytes)
 
 	cacheDir := string(cfg.MountConfig.CacheDir)
-	// Use temp-dir as default cache-dir.
-	if cacheDir == "" {
-		if cfg.TempDir == "" {
-			cacheDir = os.TempDir()
-		} else {
-			cacheDir = cfg.TempDir
-		}
-	}
-	cacheDir, err := filepath.Abs(cacheDir)
-	if err != nil {
-		panic(fmt.Errorf("createFileCacheHandler: error while resolving cache-dir (%s) in config-file: %w", cacheDir, err))
-	}
 	// Adding a new directory inside cacheDir, so that at the time of Destroy
```

Comment:
slightly unrelated to this PR, but we might want to update this comment.

Reply:
Thanks for pointing out. Corrected it.